dao/src/integ-test/java/uk/gov/moj/sdt/dao/TargetApplicationDaoTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.moj.sdt.dao;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.moj.sdt.dao.api.ITargetApplicationDao;
+import uk.gov.moj.sdt.dao.config.DaoTestConfig;
+import uk.gov.moj.sdt.domain.TargetApplication;
+import uk.gov.moj.sdt.domain.api.ITargetApplication;
+import uk.gov.moj.sdt.test.utils.AbstractIntegrationTest;
+import uk.gov.moj.sdt.test.utils.TestConfig;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Test {@link TargetApplicationDao} query methods.
+ *
+ * @author Mark Dathorne
+ */
+@ActiveProfiles("integ")
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = { TestConfig.class, DaoTestConfig.class})
```

Comment:
Are both of these annotations needed?  I've seen test classes that just use the SpringBootTest annotation.

Reply:
Resolved